xbmc/interfaces/json-rpc/PlayerOperations.cpp

```diff
@@ -489,6 +489,30 @@ JSONRPC_STATUS CPlayerOperations::Zoom(const std::string &method, ITransportLaye
   }
 }
 
+JSONRPC_STATUS CPlayerOperations::ViewMode(const std::string &method, ITransportLayer *transport, IClient *client, const CVariant &parameterObject, CVariant &result)
+{
+  CVariant viewMode = parameterObject["viewmode"];
+  CVariant zoom = parameterObject["zoom"];
+  CVariant pixelRatio = parameterObject["pixelratio"];
+  CVariant verticalShift = parameterObject["verticalshift"];
+  CVariant stretch = parameterObject["stretch"];
+
+  if (viewMode.isInteger() &&
+      zoom.isDouble() &&
+      pixelRatio.isDouble() &&
+      verticalShift.isDouble() &&
+      stretch.isBoolean())
+  {
+    g_application.GetAppPlayer().SetRenderViewMode((int)viewMode.asInteger(),
+                                                   (double)zoom.asDouble(),
+                                                   (double)pixelRatio.asDouble(),
+                                                   (double)verticalShift.asDouble(),
+                                                   (bool)stretch.asBoolean());
```

Comment:
`asBoolean` returns a `bool`, no need to cast
```suggestion
                                                   stretch.asBoolean());
```